assets/stylesheets/base/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin border-bottom($width: 1px, $style: solid, $color: $underline-color) {
+  border-bottom: $width $style $color;
+}
+
+@mixin alert($background, $border, $text) {
```

Comment:
Likewise – not convinced that this providing enough value to warrant its own global mixin. We're only using this in one place within this package; I'd argue that we should only create mixins when we need them and not try to optimize this code prematurely.